.github/workflows/build.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
-        runner: [windows-large, macos-14-xlarge]
+        runner: [windows-large, macos-15-xlarge]
```

Comment:
Thanks! That was the initial plan for the xcode16 update, but there were no available runners at the time.